todo-app.css

```diff
@@ -0,0 +1,90 @@
+:root {
+  --purple-color: rgba(112, 35, 112, 1);
+  --gray-color: rgb(120, 120, 120);
+  --light-purple-color: rgba(238, 212, 238, 1);
```

Comment:
variable should be more like `heading-color`, `border-color` so that we know what variable is for what